ledger/participant-state-errors/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Nit:
To signal that this is a KV specific module, you might want to name it as `participant-state-kv-errors`.
Moving these error definitions to under a common root with the other error documentation related definitions such as `com.daml.ledger.error.definitions.kv.*` may also make sense (however, then the module name/location would need to change as well).